src/tests/unit/tests/electron/platform/android/__snapshots__/scan-results-to-unified-results.test.ts.snap

```diff
@@ -4,43 +4,86 @@ exports[`ScanResultsToUnifiedResults Null ScanResults input returns empty output
 
 exports[`ScanResultsToUnifiedResults ScanResults with no RuleResults returns empty output 1`] = `Array []`;
 
-exports[`ScanResultsToUnifiedResults ScanResults with passes, failures, and excluded results 1`] = `
+exports[`ScanResultsToUnifiedResults ScanResults with passes, failures, view IDs, and excluded results 1`] = `
 Array [
   Object {
-    "descriptors": null,
+    "descriptors": Object {
+      "boundingRectangle": Object {
+        "bottom": 800,
+        "left": 10,
+        "right": 600,
+        "top": 0,
+      },
+      "className": "myClass1",
+      "contentDescription": "myDescription1",
+      "text": "myText1",
+    },
```

Comment:
@karanbirsingh, please let me know if there are other properties you think we ought to have in the descriptor properties

Reply:
I think these are good